dali/operators/signal/wavelet/cwt_op_gpu.cu

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2019-2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <memory>
+#include <utility>
+#include <vector>
+#include "dali/core/static_switch.h"
+#include "dali/kernels/kernel_manager.h"
+#include "dali/kernels/kernel_params.h"
+#include "dali/kernels/signal/wavelets/cwt_args.h"
+#include "dali/kernels/signal/wavelets/cwt_gpu.h"
+#include "dali/operators/signal/wavelets/cwt_op.h"
+#include "dali/pipeline/data/views.h"
+
+namespace dali {
+
+DALI_SCHEMA(Cwt).DocStr("by MW").NumInput(1).NumOutput(1).AddArg("a", "costam",
+                                                                 type2id<float>::value);
+
+template <typename T>
+struct CwtImplGPU : public OpImplBase<GPUBackend> {
+ public:
+  using CwtArgs = kernels::signal::wavelets::CwtArgs<T>;
+  using CwtKernel = kernels::signal::wavelets::CwtGpu<T>;
+
+  explicit CwtImplGPU(CwtArgs args) : args_(std::move(args)) {
```

Comment:
I wonder if this is a parameter that you want to set once for all or it could differ sample to sample.